one.txt

```diff
@@ -1 +1 @@
-This is the first file
+OVERWRITTEN FORCEFULLY
```

Comment:
A comment

Reply:
Review comment in thread mention https://github.com/yamiacat/MoreTestRenamed/pull/32